src/d3a/models/strategy/__init__.py

```diff
@@ -402,13 +402,25 @@ def posted_bid_energy(self, market_id):
             return 0.0
         return sum(b.energy for b in self._bids[market_id])
 
-    def remove_bid_from_pending(self, bid_id, market_id):
+    def remove_bid_from_pending(self, market_id, bid_id=None):
```

Comment:
I assume the reversal of the  argument order is not really necessary, mostly cosmetics for matching the order of the EventMixin methods argument order (the market precedes the other arguments in these methods IIRC), or am I mistaken?

Reply:
I wanted to be explicit and emphasise with the optional argument "bid" that there is an option to dont parse it or set it to None. For that i had to change the order.